llvm/tools/llvm-dwarfdump/llvm-dwarfdump.cpp

```diff
@@ -836,6 +842,8 @@ int main(int argc, char **argv) {
   }
   if (!Verify && ErrorDetails != Unspecified)
     WithColor::warning() << "-error-detail has no affect without -verify";
+  if (!Verify && !JsonSummaryFile.empty())
+    WithColor::warning() << "-json-summary-file has no affect without -verify";
```

Comment:
Should these flags imply `-verify`?

Reply:
I'm happy to flip both of them to imply -verify. It definitely makes usage easier.

Reply:
He might be asking if we just need to supply `--json-summary-file` and that would enable verify automatically? But I would like it to be clear that this flag goes with `--verify`. Maybe `--json-verify-results-file`?

Or can we have optional values for the existing `--verify` so it would be:
```
--verify
--verify=text
--verify=json
```
Where `--verify` takes an optional enum parameter: `(text|json)` and it defaults to `text`? I am not sure this make sense though

Reply:
Why did I not do this with --error-details already? It's *so* much easier!

Reply:
Check the new help text.